py/desitarget/cuts.py

```diff
@@ -1192,515 +1192,139 @@ def isMWSSTAR_colors(gflux=None, rflux=None, zflux=None, w1flux=None, w2flux=Non
-    bgs &= (gfracin>0.3) & (rfracin>0.3) & (zfracin>0.3)
-    bgs &= (gfluxivar>0) & (rfluxivar>0) & (zfluxivar>0)
-    bgs &= rflux > gflux * 10**(-1.0/2.5)
-    bgs &= rflux < gflux * 10**(4.0/2.5)
-    bgs &= zflux > rflux * 10**(-1.0/2.5)
-    bgs &= zflux < rflux * 10**(4.0/2.5)
-    bgs &= ~brightstarinblob
-    bgs_gaia = bgs & (Grr > 0.6)
-    bgs_nogaia = bgs & (gaiagmag == 0)
-    bgs = bgs_gaia | bgs_nogaia
-    #if objtype is not None:
-    #    bgs &= ~_psflike(objtype)
     return bgs
 
 
-def isBGS_wise(gflux=None, rflux=None, zflux=None, w1flux=None, w2flux=None, 
-               gnobs=None, rnobs=None, znobs=None, gfracmasked=None, rfracmasked=None, zfracmasked=None,
-               gfracflux=None, rfracflux=None, zfracflux=None, gfracin=None, rfracin=None, zfracin=None,
-               gfluxivar=None, rfluxivar=None, zfluxivar=None, brightstarinblob=None, Grr=None, w1snr=None,
-               gaiagmag=None, objtype=None, primary=None, south=True):
-    """Convenience function for backwards-compatability prior to north/south split.
-
-    Args:
-        gflux, rflux, zflux, w1flux, w2flux: array_like
-            The flux in nano-maggies of g, r, z, w1, and w2 bands.
-        gnobs, rnobs, znobs: array_like or None
-            Number of observations in g, r, z bands
-        gfracmasked, rfracmasked, zfracmasked: array_like or None
-            Profile-weighted fraction of pixels masked from all observations of this object in g,r,z
-        fracflux, rfracflux, zfracflux: array_like or None
-            Profile-weighted fraction of the flux from other sources divided by the total flux in g,r,z
-        gfracin, rfracin, zfracin: array_like or None
-            Fraction of a source's flux within the blob in g,r,z
-        gfluxivar, rfluxivar, zfluxivar: array_like or None
-            inverse variance of FLUX g,r,z
-        brightstarinblob: boolean array_like or None
-            ``True`` if the object shares a blob with a "bright" (Tycho-2) star
-        Grr: array_like or None
-            Gaia G band magnitude minus observational r magnitude
-        w1snr: array_like or None
-            W1 band signal to noise
-        gaiagmag: array_like or None
-            Gaia G band magnitude
-        objtype: array_like or None
-            If given, The TYPE column of the catalogue.
-        primary: array_like or None
-            If given, the BRICK_PRIMARY column of the catalogue.
-        south: boolean, defaults to ``True``
-            Call isBGS_bright_north if south=False, otherwise call isBGS_bright_south.
-
-    Returns:
-        mask : array_like. True if and only if the object is a BGS target.
+def isBGS_colors(gflux=None, rflux=None, zflux=None, w1flux=None, w2flux=None,
+                 Grr=None, gaiagmag=None, south=True, targtype=None):
+    """Standard set of masking cuts used by all BGS target selection classes
+    (see, e.g., :func:`~desitarget.cuts.isBGS` for parameters).
```

Comment:
`desisim.templates.BGS` knows nothing about Gaia and in any case we're using the Gaia cuts for star-galaxy separation, not for color/flux cuts.  If you could push the Gaia cuts to `notin_BGS_mask` then I can use `isBGS_colors` in `desisim.templates.BGS` out of the box.

Reply:
OK, I've shifted all of the Gaia cuts to `notinBGS_mask` and checked everything is consistent with Omar's original output. 

If this looks like it's going to work with `desisim.templates.BGS`, feel free to merge.